Simplified/Catalog/NYPLCatalogGroupedFeed.m

```diff
@@ -91,6 +91,12 @@ - (instancetype)initWithOPDSFeed:(NYPLOPDSFeed *)feed
   NSMutableDictionary *const groupTitleToMutableBookArray = [NSMutableDictionary dictionary];
   NSMutableDictionary *const groupTitleToURLOrNull = [NSMutableDictionary dictionary];
   
+  // Create an expiration date object with value of 2 months from now
+  NSDateComponents *dateComponents = [[NSDateComponents alloc] init];
```

Comment:
since these 4 lines are repeated for NYPLCatalogUngroupedFeed, I was wondering if we should move them to a "createBannedBookExpiration" method in NYPLBook maybe?